app/apollo/subscription/index.js

```diff
@@ -53,6 +53,7 @@ class PubSubImpl {
   
   constructor(params) {
     this.initRetries = 0;
+    this.pubsubQueue = new Array();
```

Comment:
we probably do not need an array here, as long as the variable could store one latest {timestamp: <date-time>} should be enough.